salt/modules/firewalld.py

```diff
@@ -512,20 +532,23 @@ def add_port_fwd(zone, src, dest, proto='tcp', dstaddr=''):
         salt '*' firewalld.add_port_fwd public 80 443 tcp
     '''
     if not get_masquerade(zone):
-        add_masquerade(zone)
+        add_masquerade(zone, permanent)
 
-    return __firewall_cmd(
-        '--zone={0} --add-forward-port=port={1}:proto={2}:toport={3}:toaddr={4}'.format(
+    cmd = '--zone={0} --add-forward-port=port={1}:proto={2}:toport={3}:toaddr={4}'.format(
             zone,
             src,
             proto,
             dest,
             dstaddr
         )
```

Comment:
I think this formatting needs to be updated. The closing bracked is intented one level too much. Same on line 563.